html/objects/_flag.scss

```diff
@@ -132,15 +117,15 @@
 }
 
 @media all and (max-width: $sprk-flag-stacked-breakpoint) {
-  
+
   /// Stacked flag at smaller screen width.
   .sprk-o-Flag--stacked {
-    direction: ltr;
+    flex-direction: column;
 
-    > .sprk-o-Flag__body,
-    > .sprk-o-Flag__figure,
-    & {
-      display: block;
+    &.sprk-o-Flag--rev {
+      > .sprk-o-Flag__figure {
+        order: 0;
```

Comment:
hey @RVMendoza , Did you forget to remove this order property? Looks like it's not doing anything.